### PR TITLE
New article for season of giving 2020

### PR DESCRIPTION
**Pull request by andrewthetechie**

### CODE REVIEW

File: content/articles/SADevs-season-of-giving-2020.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Do you have an organization you like to work with? Reach out to the admins and we'll add it to our list!
+
+#### San Antonio Food Bank
+https://safoodbank.org/
+
+#### Hill Country Daily Bread
+https://hillcountrydailybread.com/
+
+####Get Up Community Center
+https://www.getupcommunity.org/food-pantry
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:35]:
```suggestion
<https://www.getupcommunity.org/food-pantry>
```

File: content/articles/SADevs-season-of-giving-2020.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+These are charities that we feel are doing good work in our community. Don't feel like you have to donate to these 
+organizations, but if you're looking for a place to donate these are good choices!
+
+Do you have an organization you like to work with? Reach out to the admins and we'll add it to our list!
+
+#### San Antonio Food Bank
+https://safoodbank.org/
+
+#### Hill Country Daily Bread
+https://hillcountrydailybread.com/
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:32]:
```suggestion
<https://hillcountrydailybread.com/>
```

File: content/articles/SADevs-season-of-giving-2020.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+reach out to the admins and we'll add it to our donation list! 
+
+### Suggested Charities
+These are charities that we feel are doing good work in our community. Don't feel like you have to donate to these 
+organizations, but if you're looking for a place to donate these are good choices!
+
+Do you have an organization you like to work with? Reach out to the admins and we'll add it to our list!
+
+#### San Antonio Food Bank
+https://safoodbank.org/
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:29]:
```suggestion
<https://safoodbank.org/>
```

File: content/articles/SADevs-season-of-giving-2020.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+Title: SA Devs Season of Giving
+Date: 11-01-2020
+Tags: season-of-giving, donations, charity
+Slug: season-of-gicing-2020
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:4]:
```suggestion
Slug: season-of-giving-2020
```

File: content/articles/SADevs-season-of-giving-2020.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+https://hillcountrydailybread.com/
+
+####Get Up Community Center
+https://www.getupcommunity.org/food-pantry
+
+
+### How to record a donation
+
+1. Take a screenshot or PDF of your donation receipt
++ Open a DM to @sadevbot
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:41]:
```suggestion
+ Open a DM to `@sadevbot`
```

File: content/articles/SADevs-season-of-giving-2020.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
++ Send ./donation {amount} --make-public and attach your receipt to the message
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:42]:
```suggestion
+ Send `./donation {amount} --make-public` and attach your receipt to the message 
```

File: content/articles/SADevs-season-of-giving-2020.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
++ Open a DM to @sadevbot
++ Send ./donation {amount} --make-public and attach your receipt to the message 
+If you want to keep your donation private (the admins will still see your name/info but we won't put it on the site) 
+leave out --make-public
+
+Make sure your amount matches the amount on your receipt. You can enter it as $##.## i.e. $20.00 (or even just $20).
++ Sadevbot will send your donation on to the admins for review and it'll get added to the website (just below here in 
+the donations list). Private donations will just list an amount. 
+
+It'll also update totals in #sadevs-season-of-giving-2020
```
Comment (gtmanfred) [inline content/articles/SADevs-season-of-giving-2020.md:50]:
```suggestion
It will also update totals in `#sadevs-season-of-giving-2020`
```

**Review by gtmanfred — APPROVED** [review]
Review app looks good to me. I assume you aren't going to publish this until Sunday?

**Comment by andrewthetechie** [comment]
@gtmanfred I was going to publish it/announce it today but not put the bot plugin live until sunday for donations.

**Review by gtmanfred — APPROVED** [review]

**Pull request merged**